e2e_tests/management/test_management.py

```diff
@@ -155,3 +159,51 @@ def test_visibility_workspace(self, environment: "Environment"):
         delete_project(name=created_project_identifier, api_token=environment.admin_token)
 
         assert project_identifier not in get_project_list(api_token=environment.admin_token)
+
+    @staticmethod
+    def _create_project(environment: "Environment", *, name_suffix):
```

Comment:
Why u need this instead of re-use of the project provided by environment?

Reply:
Good enough